ui/src/types/labels.ts

```diff
@@ -1,7 +1,7 @@
 import {Label as GenLabel} from 'src/client'
 
 export interface Label extends GenLabel {
-  properties: LabelProperties
+  properties?: LabelProperties
```

Comment:
I'm confused about this change. The point of adding label defaults is that we are taking something of `GenLabel` type which has optional and any properties, and adding default properties to ensure they are there and then it becomes a Label type with required properties type.

Reply:
That's a good point. I think it had originally been set to a required property to match with some of the expectations of the not-yet-refactored API functions, but removing it and defaulting to `export {Label} from 'src/client` seems to work fine now